pkg/kotsutil/kots.go

```diff
@@ -38,6 +38,16 @@ type KotsKinds struct {
 	Backup *velerov1.Backup
 }
 
+// KustomizeVersion will return the kustomize version to use for this application
+// applying the default, if there is one, for the current version of kots
+func (k KotsKinds) KustomizeVersion() string {
+	if k.KotsApplication.Spec.KustomizeVersion != "" {
```

Comment:
I think we should check for `latest` here

Reply:
Do we support the string "latest"?